api/controllers/apps.go

```diff
@@ -47,11 +47,13 @@ func AppShow(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 func AppCreate(rw http.ResponseWriter, r *http.Request) *httperr.Error {
 	name := r.FormValue("name")
 
-	app := &models.App{
-		Name: name,
+	app, err := models.GetApp(name)
```

Comment:
Ok I reviewed everything and I think I get it. When a user says create app "foo", you want to see if there's an unbound app "foo" before creating "convox-foo". Can you add a comment or two to this effect?